doc/ctc-meetings/2016-08-17.md

```diff
@@ -0,0 +1,300 @@
+* Seek legal advice on LICENSE and copyright blocks in code [#3979](https://github.com/nodejs/node/issues/3979)
+
+
+## Previous Meeting
+
+Extracted from **ctc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* v4.5.0 proposal [#7688](https://github.com/nodejs/node/pull/7688)
+
+  * @thealphanerd will investigate problem with Bower, may delay release, if so will notify everyone.
+
+* buffer: hard-deprecate Buffer constructor [#7152](https://github.com/nodejs/node/pull/7152)
+
+  * Continue discussion in GitHub.
+  * Finalize decision next week.
+
+* Revert "fs: add a temporary fix for re-evaluation support" [#6413](https://github.com/nodejs/node/pull/6413)
+
+  * Myles to follow up with Gulp, notify CTC as appropriate.
+  * Finalize decision next week.
+
+* errors: add internal/errors module [#6573](https://github.com/nodejs/node/pull/6573)
+
+  * Review in GitHub.
+  * Finalize next week.
+  * **Decided not to discuss today, continue work in GH.**
+
+* Introduce staging branch for stable release streams [#6306](https://github.com/nodejs/node/issues/6306)
+
+  * Continue discussion in GH.
+  * Keep on agenda.
+
+### nodejs/node-eps
+
+* proposal: WHATWG URL standard implementation [#28](https://github.com/nodejs/node-eps/pull/28)
+
+  * Remove global than land EP.
+  * Continue work/review on impl.
+
+### general
+
+* discussion: CommonJS and ES6 modules interoperability (<https://gist.github.com/bmeck/52ee45e7c34d1eac44ce8c5fe436d753>)
+
+  * Schedule a session specifically on modules.
+
+
+## Minutes
+
+### CTC membership nomination: @TheAlphaNerd [#8058](https://github.com/nodejs/node/issues/8058)
+
+Generally we require attendance as an observer at least 4 times to ensure fit on both sides. Myles has already attended several meetings and all are comfortable.
+
+We will vote on this next week.
+
+**Next steps**:
+
+* Vote next week.
+
+---
+
+### buffer: hard-deprecate Buffer constructor [#7152](https://github.com/nodejs/node/pull/7152)
+
+In order to enable inheritance from Buffer, need to declare it as a class, which will then require `new` keyword to create a new Buffer.
+
+`new Buffer(...)` works in all versions.
+
+Options
+
+1. Keep as is - `Buffer(...)` would still work without `new`, and Buffer wouldn’t be subclassable.
+2. Deprecate creating a buffer without `new`. `Buffer(...)` would no longer work.
+3. Fully deprecate any use of Buffer constructor even without `new`. Neither `Buffer(...)` nor `new Buffer(...)` would work.
+
+@trevnorris: Another option is to retain UInt8Array constructor signatures as Buffer constructor signatures.
+
+@Fishrock123: We should deprecate use without `new`, but deprecating entire constructor isn’t reasonable due to ecosystem usage.
+
+@jasnell: It would be a hard deprecation warning, i.e. throw if used without `new`.
+
+@trott: That would imply full removal in v9.x.
```

Comment:
That would imply full removal in v9.x *at the earliest*. (It's likely I wasn't very clear in my wording and/or was talking way too fast for anyone to understand me, since both of those are things I do any time I open my mouth.)